src/device_impl.rs

```diff
@@ -102,6 +106,23 @@ where
         Ok(())
     }
 
+    /// Set the accelerometer FIFO mode and full threshold (0 - 31).
```

Comment:
It would be worth documenting here that a fth > 31 is automatically capped to 31